Resources/Prototypes/_CM14/Roles/Jobs/Marines/hospital_corpsman.yml

```diff
@@ -36,17 +34,17 @@
 - type: startingGear
   id: HospitalCorpsmanGear
   equipment:
-    jumpsuit: ClothingUniformJumpsuitBrigmedic
-    outerClothing: ClothingOuterCoatAMG
+    jumpsuit: CMJumpsuitMarineMedic
     back: ClothingBackpackBrigmedicFilled
-    shoes: ClothingShoesColorRed
-    gloves: ClothingHandsGlovesNitrile
+    shoes: CMShoesCombatBoots
+    gloves: CMHandsBlack
     eyes: ClothingEyesHudMedical
-    head: ClothingHeadHatBeretBrigmedic
+    head: CMArmorHelmetM10Medic
+    outerClothing: ArmorMarineM3
     id: DogtagHospitalCorpsman
     ears: ClothingHeadsetBrigmedic
     belt: ClothingBeltMedicalFilled
```

Comment:
This needs to be updated btw

Reply:
We don't have pouches and belts from cm13 yet, so i just left it be